### PR TITLE
WEB-54: update standard card title font and proxima nova font-face

### PR DESCRIPTION
**Pull request by oddwili**
This also removes old comments to take out blocks after brand consolidation wrapped up.

This should be reviewed alongside: https://github.com/Americastestkitchen/espresso/pull/4691

### CODE REVIEW

File: src/styles/typography.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ const fontSize = {
 const font = {
   clb: 'Clarendon Bold,serif',
   mwr: 'Merriweather, serif',
-  pnb: 'proximaNovaBold, sans-serif',
+  pnb: 'proxima-nova, sans-serif',
   pnr: 'proximaNovaRegular, sans-serif',
```
Comment (harry-atk) [inline src/styles/typography.ts:16]: should be proxima-nova to match the change to font-face

File: src/components/Cards/shared/Title/index.js [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
 import PropTypes from 'prop-types';
 import styled, { css } from 'styled-components';
-import { color, font, fontSize, lineHeight, withThemes } from '../../../../styles';
+import { color, fontSize, lineHeight, withThemes } from '../../../../styles';
 
 const TitleTheme = {
   default: css`
-    font: ${fontSize.xl}/${lineHeight.sm} ${font.pnb};
+    font: ${fontSize.xl}/${lineHeight.sm} proxima-nova, sans-serif;
+    font-weight: 700;
```
Comment (joshuabrileyatk) [inline src/components/Cards/shared/Title/index.js:9]: Not a deal breaker, but the font weight can also be included as part of the font shorthand line above. 
https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties#font_properties

**Review by joshuabrileyatk — APPROVED** [review]

File: src/styles/typography.ts (hunk 1 above)
Comment (harry-atk) [reply src/styles/typography.ts:16]: also a lot of pnb usage we need font weights for https://github.com/search?q=repo%3AAmericastestkitchen%2Fmise-ui+pnb&type=code

File: src/components/Accordion/index.js [hunk 3]
```diff
@@ -38,7 +38,8 @@ const AccordionButtonTheme = {
     align-items: center;
     border: none;
     display: flex;
-    font: ${fontSize.md}/1 ${font.pnb};
+    font: ${fontSize.md}/1 ${font.pnb} 700;
+    font-weight: 700;
```
Comment (joshuabrileyatk) [inline src/components/Accordion/index.js:42]: Line 42 seems to be redundant, since we're adding the font weight in the font shorthand line above.

**Review by joshuabrileyatk — COMMENTED** [review]
Looks good. One small change request.

**Review by harry-atk — CHANGES_REQUESTED** [review]
want to confirm the font shorthand works first

**Comment by oddwili** [comment]
closing in favor of https://github.com/Americastestkitchen/mise-ui/pull/999

**Pull request closed without merging**